web/template/layout/header.gohtml

```diff
@@ -1,45 +1,91 @@
+            <image src="{{ prefix "/assets/images/moj-logotype-crest.png" }}" xlink:href="" class="moj-header__logotype-crest-fallback-image" width="40" height="40"></image>
+          </svg>
+          <a class="moj-header__link moj-header__link--organisation-name" href="#">OPG</a>
```

Comment:
Since this doesn't go anywhere, probably best to make it a span:
```suggestion
          <span class="moj-header__link moj-header__link--organisation-name">OPG</span>
```